src/components/PanelV2/PanelV2.stories.js

```diff
@@ -125,13 +175,31 @@ storiesOf(patterns('PanelV2'), module)
                   label: text('closeButton.label', closeButtonLabel),
                 }}
                 renderFooter={() => (
-                  <Button
-                    id="p2_primary-button"
-                    size="large"
-                    onClick={this.closeSecond}
-                  >
-                    Add
-                  </Button>
+                  <ComboButton {...comboButtonProps()}>
+                    <ComboButtonItem
+                      onClick={action('onClick (Item 1 - primary button)')}
+                      renderIcon={ArrowRight20}
+                    >
+                      Item 1 (becomes primary button and text will be truncated)
```

Comment:
Should we do this as a template string { `Item...` }

Reply:
Sure thing -- this was autoformatting from prettier so I don't mind either way 👍